recipes-asteroid/qml-asteroid/qml-asteroid_git.bb

```diff
@@ -11,34 +11,17 @@ PV = "+git${SRCPV}"
 S = "${WORKDIR}/git"
 inherit qmake5
 
-PACKAGE_ARCH = "${MACHINE_ARCH}"
```

Comment:
But this is still machine specific :thinking:

Reply:
I can't remember whether there is a way to specify that only a subpackage, say, qml-asteroid-config, is machine specific.
Maybe you could also create a separate recipe for the machine config because even though qml-asteroid will be its main (only) user for now, it makes sense for other components to look into this file. And that recipe would have PACKAGE_ARCH = "${MACHINE_ARCH}"